mikeio/dfsu.py

```diff
@@ -1149,6 +1125,101 @@ def get_node_centered_data(self, data, extrapolate=True):
 
         return node_centered_data
 
+    def _Get_2DVertical_elements(self):
+        if (self._type == DfsuFileType.DfsuVerticalProfileSigmaZ) or (
+            self._type == DfsuFileType.DfsuVerticalProfileSigma
+        ):
+            elements = [
+                list(self._source.ElementTable[i])
+                for i in range(len(list(self._source.ElementTable)))
+            ]
+            return np.asarray(elements) - 1
+
+    def plot_vertical_profile(
```

Comment:
Would it make sense to create a new specialized class only for vertical slices, and rename this method to plot?

Reply:
Yes, good idea

Reply:
Then no matter what dfsu file you open, you have a relevant plot() function - we should then change a few things in the original dfsu.plot() e.g. rename first argument to values